setup/monitoring/stackdriver.md

```diff
@@ -6,3 +6,92 @@ sidebar:
 ---
 
 {% include toc %}
+
+There are two ways to use Stackdriver with Spinnaker. The easiest is
+to configure Spinnaker microservices to push metrics directly into
+stackdriver. Additionally the Spinnaker Monitoring Daemon can write
+into Stackdriver for you.
+
+Note that Spinnaker talks to Stackdriver via HTTP so you do not need
+to install the Stackdriver Agent. Since the Agent is only available on
+Google Cloud Platform or Amazon Web Services, this means that you can
+write metrics into Stackdriver regardless of what platform you are running
+Spinnaker on. You may still want to install a Stackdriver Agent for
+local machine monitoring or monitoring other services. However, the
+agent itself does not know about Spinnaker and will not monitor it.
+
+*__Warning__: The current integration with Stackdriver conflicts with
```

Comment:
While true, I highly suggest rewording this to be less...incendiary. Consider:

"The current integration uses Stackdriver custom metrics, which are the most flexible way to integrate the dynamic number of time-series generated by each of Spinnaker's microservices. As a result, even a single VM deployment can cause hundreds of dollars of charges per month. If you are interested in using Stackdriver, please contact us through the Spinnaker Slack channel"

@duftler